core/src/main/kotlin/ch/ergon/dope/resolvable/clause/model/setoperators/SetOperator.kt

```diff
@@ -0,0 +1,23 @@
+package ch.ergon.dope.resolvable.clause.model.setoperators
+
+import ch.ergon.dope.DopeQuery
+import ch.ergon.dope.DopeQueryManager
+import ch.ergon.dope.resolvable.clause.ISelectOffsetClause
+
+sealed class SetOperator(
+    private val symbol: String,
```

Comment:
Maybe go for an enum with a string-representation rather than a simple string

Reply:
done